packages/docs/docs/tutorial-basics/events.md

```diff
@@ -56,6 +56,48 @@ export default function MyButton() {
 }
 ```
 
+## Privacy friendly
+
+Syft makes it easy to handle user privacy preferences. This functionality needs to be enabled explicitly as shown below.
+
+```jsx title="src/pages/_app.tsx"
+// ...
+<SyftProvider gdpr={{ enabled: true }} />
```

Comment:
instead of gdpr, call it consent? gdpr is not the only privacy standard I believe that requires it.

Reply:
consent.enabled may mean that consent is given.  can you recommend another word to indicate that this is about consent-management.